util.c

```diff
@@ -8,7 +8,7 @@
 #include <sys/stat.h>
 #include "util.h"
 
-extern char *argv0;
```

Comment:
This change breaks OpenBSD right?

Simple enough to rename this variable though